ru.sview.sView.json

```diff
@@ -0,0 +1,141 @@
+{
+    "app-id" : "ru.sview.sView",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "21.08",
+            "add-ld-path": "."
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "command" : "sView",
+    "rename-desktop-file" : "sViewMP.desktop",
+    "rename-icon" : "sview",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* GPU acceleration */
+        "--device=dri",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Get access to the files */
+        "--filesystem=host:ro",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfs:ro"
+    ],
+    "modules" : [
+        "shared-modules/gtk2/gtk2.json",
+        {
+            "name" : "libconfig",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz",
+                    "sha256": "7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9"
+                }
+            ]
+        },
+	{
+	    "name": "wxWidgets",
+	    "rm-configure": true,
+            "cleanup": [
+                "/bin",
+                "/lib/wx",
+                "/share/bakefile"
+            ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
```

Comment:
does this explicitly not work with 3.0.5 then? I know there's only a few bug fixes in there, just a slight surprise that's all.

Reply:
It won't be such a surprise when you look at the commit log ;)